src/Context/BasicContext.php

```diff
@@ -1290,4 +1289,38 @@ protected function logException(Exception $exception)
     {
         file_put_contents('php://stderr', 'Exception caught: ' . $exception->getMessage());
     }
+
+    /**
+     * Detect element with javascript, rather than having the selector converted to xpath
+     * There's already an xpath based function 'I see the "" element' iSeeTheElement() in silverstripe/cms
+     * There's also an 'I should see "" element' in MinkContext which also converts the css selector to xpath
+     *
+     * @When /^I should see the "([^"]+)" element/
+     * @param $selector
+     */
+    public function iShouldSeeTheElement($selector)
+    {
+        $sel = str_replace('"', '\\"', $selector);
+        $js = <<<JS
+return document.querySelector("$sel");
+JS;
+        $element = $this->getSession()->evaluateScript($js);
+        assertNotNull($element, sprintf('Element %s not found', $selector));
+    }
```

Comment:
Don't think this needed anymore

Reply:
As noted [here](https://github.com/silverstripe/silverstripe-asset-admin/pull/1218#discussion_r682123327) we require this

Reply:
Seems like we don't need this for this bit of work. I'd like to keep this one regardless though because it's a very useful utility function since it uses css selectors instead of css selector conversted to xpath